lib/index.d.ts

```diff
@@ -45,6 +45,27 @@ export declare var scenario: (name: string) => ScenarioDeclaration;
  */
 export declare var step: (stepCaption: string, additionalProperties?: StepProperties) => any;
 
+/**
+ * Saves a step in your e2e tests.
+ *
+ * Use this decorator (http://www.typescriptlang.org/docs/handbook/decorators.html) this in your e2e test functions
+ * whenever you want scenarioo to report a step based on a call of a function on a page object.
+ *
+ * @param {string} [stepDescription?] - optional description text for the step to be recorded, will be displayed in `title` field of a step in scenarioo.
+ * if not provided, it will use the following pattern: `{objectName}: {methodName}`.
+ */
+export declare var stepAnnotation: (stepDescription?: string) => any;
```

Comment:
would be cool if also additionalProperties could be pased, where necessary, example if I want to put labels on a step,as in the usual step function